### PR TITLE
(SECURITY) When "Add" permission is selected then "view" should not b…

### PR DESCRIPTION
**Pull request by rajeshsilverxis**
…e unselected

### CODE REVIEW

File: src/app/components/user-role/edit-user-roles.component.ts [hunk 1]
```diff
@@ -422,6 +437,7 @@ export class EditUserRolesComponent implements OnInit {
     }
 
     hasPages(currentModule: ModuleHierarchyMaster, type: string, checkedValue: boolean) {
+
```
Comment (vishalsutharsilverxis) [inline src/app/components/user-role/edit-user-roles.component.ts:440]: Why need to add new line here ?

File: src/app/components/user-role/edit-user-roles.component.ts [hunk 2]
```diff
@@ -485,6 +501,7 @@ export class EditUserRolesComponent implements OnInit {
     }
 
     UpdateUserRole(): void {
+
```
Comment (vishalsutharsilverxis) [inline src/app/components/user-role/edit-user-roles.component.ts:504]: Why need to add new line here ?

File: src/app/components/user-role/edit-user-roles.component.ts [hunk 3]
```diff
@@ -524,6 +541,7 @@ export class EditUserRolesComponent implements OnInit {
     }
 
     checkparent(currentModule,parentId,type){        
+
```
Comment (vishalsutharsilverxis) [inline src/app/components/user-role/edit-user-roles.component.ts:544]: Why need to add new line here ?

**Comment by rajeshsilverxis** [comment]
I have solved When "Add" permission is selected then "view" should not be allow unchecked in security

**Pull request merged**